src/boards/FRM301.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
This is same target as `stm32f030c8.json`. These board files can be merged to avoid any extra maintenance.

Reply:
Thanks  @cruwaller for the feedback. I deleted the redundant file.